docs/build/program/run.md

```diff
@@ -64,9 +64,17 @@ flutter run <DART_FILE>
 
 This is useful because as long as that computer is able to establish a network connection with the machine's computer, your control logic will be executed on the machine.
 
-## Run Code On-Machine
+## Run code on local network
 
-In case you run [PID control loops](https://en.wikipedia.org/wiki/PID_controller) or your machines have intermittent network connectivity, you can ensure this does not interfere with the code's execution, by running the control code on the same board that is running `viam-server`.
+Your machines do not need to be connected to the Internet for you to be able to run code.
+As long as your machine is connected to the same LAN or WAN network as the device running the code, you can connect to it and are run code. 
```

Comment:
"you" and "it" are ambiguous here
```suggestion
You can run code from any computer connected to the same LAN or WAN network as your {{< glossary_tooltip term_id="machine" text="machine" >}}.
```
If you don't like ^ this suggestion, be sure to fix typo:
```suggestion
As long as your machine is connected to the same LAN or WAN network as the device running the code, you can connect to it and run code. 
```